.github/workflows/theia-zowe-explorer-ci.yml

```diff
@@ -12,14 +12,22 @@ jobs:
       - uses: actions/checkout@v2
 
       # install yarn
-      - run: npm install -g yarn
+      # Don't think this is necessary, GH Actions docs says
+      # yarn is installed by default:
+      # https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs#installing-dependencies
+      # - run: npm install -g yarn
```

Comment:
do not remove this. Just add sudo:  `sudo npm install -g yarn`

Reply:
@jellypuno I will remove it, but could you explain why? Are we not using GH-hosted runners? Those have yarn installed by default apparently.